src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -86,20 +86,17 @@ object Boot extends App with LazyLogging {
 
     val executionServiceConfig = conf.getConfig("executionservice")
     val submissionTimeout = toScalaDuration(executionServiceConfig.getDuration("workflowSubmissionTimeout"))
-    val executionServiceServers = executionServiceConfig.getObject("servers").mapValues(_.unwrapped.toString)
+
+    val executionServiceServers: Map[ExecutionServiceId, ExecutionServiceDAO] = executionServiceConfig.getObject("servers").map {
+        case (strName, strHostname) => (ExecutionServiceId(strName)->new HttpExecutionServiceDAO(strHostname.unwrapped.toString, submissionTimeout))
+      }.toMap
```

Comment:
Cromwell instances are keyed by ExecutionServiceId, which constructs from a String. I'm doing this for type safety and code clarity, but there's no functional reason; it could be a String.